design/biav2-design.md

```diff
@@ -34,6 +34,7 @@ message ExecuteResponse {
     bytes item = 1;
     repeated generated.ResourceIdentifier additionalItems = 2;
     string operationID = 3;
+    repeated generated.ResourceIdentifier itemsToUpdate = 4;
```

Comment:
My original thought was `itemsToUpdateAfterOperations` but that felt too unwieldy. Another option would be `itemsToFinalize` or `postOperationItems` I'm fine with any of these, though -- lets discuss next week for a possible follow-on change. If we're going to change it, it needs to be before 1.11 is out. @shubham-pampattiwar  any thoughts here?

Reply:
Thanks @sseago I would vote for `postOperationItems` if we see them as a second list of items to backup.  
There are some inconsistencies I mentioned in 
https://github.com/vmware-tanzu/velero/pull/5849#issuecomment-1455328366

Maybe we should refine the flow here and treat the items in `itemsToUpdate` or `postOperationItems` differently.

Reply:
@reasonerjt postOperationItems works for me. I'll update that in the follow-on PR. As for the inconsistencies between this and additional items, see comments below. I think we can converge things a bit more and minimize the inconsistency -- at this point, I think the only thing that must be different is that postOperationItems are not expected to create asynchronous operations, since that would potentially lead to never-ending backup.